tests/e2e/pet.list.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect, Page } from "@playwright/test";
+import { Authenticator } from "../utils/authenticator";
+import { PetListPage } from "../pages/pet.list.page";
+import { HomePage } from "../pages/home.page";
+import data from "../properties/data.json";
+
```

Comment:
Please notice that when you are trying to list all pets in this test, there are no pets registered, therefore the test will fail. I suggest to add this test between `"should registrer a pet"` and `"should delete a pet"`, please take a look at: [pet.registration.spec.ts](https://github.com/josdem/playwright-vetlog/blob/main/tests/e2e/pet.registration.spec.ts)